tests/AssetManagerTest/Service/AssetManagerTest.php

```diff
@@ -208,7 +209,63 @@ public function testSetExtensionFilters()
         $assetManager->setAssetOnResponse($response);
         $this->assertEquals($minified, $response->getBody());
     }
+    public function testSetExtensionFiltersNotDuplicate()
```

Comment:
This test is too complicated/hard to read. Reduce it or split it into multiple tests